wcfsetup/install/files/lib/data/article/category/ArticleCategoryNode.class.php

```diff
@@ -28,12 +28,7 @@ class ArticleCategoryNode extends CategoryNode
      */
     protected $articles;
 
-    /**
-     * Returns number of articles in the category.
-     *
-     * @return  int
-     */
-    public function getArticles()
```

Comment:
Should the old method be preserved with a `public function getArticles() { return $this->getItems(); }` implementation + deprecation? Genuine question, I don't know if this was used anywhere.